File: JavaScript Fundamentals/PointInACircle.js
function solve(args) {
    let x = Number(args[0]);
    let y = Number(args[1]);
    let distance = Math.sqrt(Math.pow(x, 2) + Math.pow(y, 2)).toFixed(2);

    if (distance > 2 || distance < -2) {

        console.log("no " + distance);
    }
    else {
        console.log("yes " + distance);
    }
}
solve(['1.5', '-1']);

File: C# I/03.Operators-And-Expressions/07.PointInACircle/PointInACircle.cs
using System;
class PointInACircle
{
    static void Main()
    {
        double x = double.Parse(Console.ReadLine());
        double y = double.Parse(Console.ReadLine());
        double distance = Math.Sqrt(Math.Pow(x, 2) + Math.Pow(y,2));
        if (distance <= 2)
        {
            Console.WriteLine("yes {0:F2}", distance);
        }

        else
        {
            Console.WriteLine("no {0:F2}", distance);
        }
    }
}
